sdk/src/main/java/com/mapbox/maps/renderer/MapboxRenderThread.kt

```diff
@@ -177,8 +178,21 @@ internal class MapboxRenderThread : Choreographer.FrameCallback {
       return
     }
     mapboxRenderer.onDrawFrame()
+    // assuming render event queue holds user's runnables with OpenGL ES commands
+    // it makes sense to execute them after drawing a map but before swapping buffers
+    if (renderEventQueue.isNotEmpty()) {
+      renderEventQueue.forEach {
+        it.runnable?.run()
+      }
+      renderEventQueue.clear()
+    }
```

Comment:
Can make `renderEventQueue` the same style as `snapshotQueue`?

Reply:
sadly no as they have different type (Runnable vs RenderEvent)